anyway_etl/waze/utils/add_time.py

```diff
@@ -0,0 +1,7 @@
+import pandas as pd
+from datetime import datetime
+
+
+def add_now_column(df: pd.DataFrame, column_name: str):
+    now = datetime.now()
+    df[column_name] = now
```

Comment:
having a module for just a single line of code makes the code much less readable. Please just write the code where it's needed

```
df[column_name] = datetime.now()
```